src/main/java/com/vinylteam/vinyl/service/impl/DefaultUserService.java

```diff
@@ -12,14 +12,13 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
-import org.elasticsearch.cli.UserException;
 import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.dao.DuplicateKeyException;
+import org.springframework.security.authentication.BadCredentialsException;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.Optional;
-import java.util.UUID;
 
 @Slf4j
 @RequiredArgsConstructor
```

Comment:
please remove 
//securityService.emailFormatCheck(email); line at all
Boyscout rule + YAGNI

plus
line 45+46 should be one line 
User userToAdd = securityService.createUserWithHashedPassword(email, password.toCharArray());